Sources/App/Core/ErrorMiddleware.swift

```diff
@@ -24,9 +24,8 @@ public final class ErrorMiddleware: AsyncMiddleware {
     public func respond(to req: Request, chainingTo next: AsyncResponder) async throws -> Response {
         do {
             return try await next.respond(to: req)
-        } catch {
-            let abortError = error as? AbortError ?? Abort(.internalServerError)
-            let statusCode = abortError.status.code
+        } catch let error as AbortError where error.status.code >= 400 {
```

Comment:
This part we'll definitely want to add: right now we're showing our _error page_ when we're doing a re-direct.

I haven't found any redirects in our current Swift code, so that shouldn't be an actual issue yet. I'm not sure if JS redirects would be affected by this but I've also not ever seen our error page with a 301 so if it is an actual issue it's quite rare.